webapp/src/pages/Settings.tsx

```diff
@@ -53,8 +60,8 @@ const FIELDS: Record<
   max_delta_representation: PERCENTAGE,
   max_delta_mean_words: { ...FLOAT, units: "words" },
   max_delta_std_words: { ...FLOAT, units: "words" },
-  short_utterance_max_word: { ...INT, units: "words" },
-  long_utterance_min_word: { ...INT, units: "words" },
+  short_sentence_max_word: { ...INT, units: "words" },
+  long_sentence_min_word: { ...INT, units: "words" },
```

Comment:
I think you did a mistake while resolving conflicts with `main`.
```suggestion
  short_utterance_max_word: { ...INT, units: "words" },
  long_utterance_min_word: { ...INT, units: "words" },
```

Reply:
I had no conflict showing for it. But, Yeah, I noticed the API response field is different from the UI, so I manually changed it.

Reply:
Restart the back end to get the latest changes.